images/fedora.yaml

```diff
@@ -140,6 +180,16 @@ packages:
       - sssd-client
      action: remove
 
+   - packages:
+     - grub2-efi-x64
+     - kernel
+     - shim
+     - btrfs-progs
```

Comment:
Hmm, why btrfs?

Reply:
IIRC something complained about btrfs missing.

Reply:
I'll test this again.

Reply:
I removed it. I had a reason for adding it but I don't remember why. Anyway, it works fine without it.